Source/WebCore/CMakeLists.txt

```diff
@@ -2837,6 +2837,11 @@ list(APPEND WebCore_PRIVATE_FRAMEWORK_HEADERS
 
     platform/graphics/InbandTextTrackPrivate.serialization.in
 
+    platform/graphics/gbm/DMABufColorSpace.serialization.in
+    platform/graphics/gbm/DMABufFormat.serialization.in
+    platform/graphics/gbm/DMABufObject.serialization.in
+    platform/graphics/gbm/DMABufReleaseFlag.serialization.in
```

Comment:
These files preferrably should go to `WebKit/Shared/glib/` instead of `WebCore`. Also, you could just put all DMABuf definitions in a single file to avoid clutter (there is already one, merging that one too into a new `ArgumentCodersDMABuf.serialization.in`.

Reply:
Does this also apply to .h?

Reply:
Not necessarily. I don't think you need to change those.